ext/gd/gd.c

```diff
@@ -1589,6 +1600,14 @@ PHP_FUNCTION(imagecreatefromstring)
 			RETURN_FALSE;
 #endif
 
+#ifdef HAVE_GD_AVIF
```

Comment:
Missing a `case PHP_GDIMG_TYPE_AVIF:` above here?

Reply:
thanks for catching that. I haven't been working with this part of the code yet, so I didn't notice it wouldn't work 🙄